.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+jobs:
+  build:
+    working_directory: /go/src/github.com/observatorium/configuration
+    docker:
+      - image: quay.io/coreos/jsonnet-ci
+    steps:
+      - checkout
+      - run: find . -type f -not -path './vendor/*' \( -name '*.libsonnet' -o -name '*.jsonnet' \) | xargs -L1 jsonnetfmt --test
+      - run: curl -o ./jb -L "https://github.com/jsonnet-bundler/jsonnet-bundler/releases/download/v0.2.0/jb-$(go env GOOS)-$(go env GOARCH)" && chmod +x ./jb
```

Comment:
This works for now :+1:  we should probably introduce a Makefile soon to encapsulate all of this logic and have it less spread out